test/common/config/new_grpc_mux_impl_test.cc

```diff
@@ -169,6 +169,24 @@ TEST_F(NewGrpcMuxImplTest, ConfigUpdateWithNotFoundResponse) {
   response->mutable_resources()->at(0).add_aliases("prefix/domain1.test");
 }
 
+// DeltaDiscoveryResponse that comes in response to an on-demand request that contain empty resource
+// list will be ignored.
+TEST_F(NewGrpcMuxImplTest, IgnoreEmptyResponse) {
```

Comment:
I think we need a test along the lines of:
 - sets up a couple of RDS' with VHDS subscriptions over ADS
 - and then starts subscriptions all at once (subscriptions should not receive responses before all of them have started) 
 - check how subscriptions handle responses (possibly check the order of responses different from the order in which requests have been sent)

That would show that multiplexing VHDS subscriptions over ADS is indeed possible.

Reply:
ok，I will try to add this test.

Reply:
sorry for no progress.  I found that empty response can not trigger init_target_ in vhds so it will block vhds initialize until time out（default timeout is 15s）, and I found after vhds update in ads，the  integration test  report an error but in out test environment， it works well.  so I need more
 time to investigate this